packages/server/tests/acceptance/index.spec.ts

```diff
@@ -132,17 +132,17 @@ describe('RPC Server Acceptance Tests', function () {
 
     function runLocalHederaNetwork() {
         // set env variables for docker images until local-node is updated
-        process.env['NETWORK_NODE_IMAGE_TAG'] = '0.35.0-alpha.6';
-        process.env['HAVEGED_IMAGE_TAG'] = '0.35.0-alpha.6';
-        process.env['MIRROR_IMAGE_TAG'] = '0.75.0-rc3';
+        process.env['NETWORK_NODE_IMAGE_TAG'] = '0.36.0-alpha.1';
+        process.env['HAVEGED_IMAGE_TAG'] = '0.36.0-alpha.1';
+        process.env['MIRROR_IMAGE_TAG'] = '0.76.1';
 
         console.log(`Docker container versions, services: ${process.env['NETWORK_NODE_IMAGE_TAG']}, mirror: ${process.env['MIRROR_IMAGE_TAG']}`);
 
         console.log('Installing local node...');
         shell.exec(`npm install @hashgraph/hedera-local -g`);
 
         console.log('Starting local node...');
-        shell.exec(`hedera start -d`);
+        shell.exec(`hedera start -d --turbo`);
```

Comment:
Perhaps not relevant to this change.  I see the option --turbo is listed in the README as "to enable/disable turbo mode."  Is turbo mode described in more detail somewhere.  I assume turbo means the local node runs faster, but what is being sacrificed to get the speed?

Reply:
Yes, it refers to a faster mode in the local-node. No sacrifice, we just skip the usage of minio and uploaders. This saves some time on exporting and importing record files. We have a issue in the local-node to add docs with diagrams about this change, I will soon start working on it.